### PR TITLE
French translation overview

### PR DESCRIPTION
**Pull request by AkimParis**
Observations:
- Most modern NLP practitioners use English words in French, so I kept them for the datasets, size of LLM (small, base, large), license, and popular research papers for this translation
- For the title, I wrote "Aperçu des grands modèles de langage (LLM) en japonais" but maybe "Aperçu des LLMs en japonais" is better to avoid brackets.
- I am not sure about the hyperlinks between languages ... : )

### CODE REVIEW

**Comment by kaisugi** [comment]
Oh, that was quick!
I'll check this PR in a few days :)

File: README_fr.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+# Aperçu des grands modèles de langage (LLM) en japonais
+[ Français | **Anglais**(./README_en.md) | [**Japonais**](./README.md) ]
```
Comment (kaisugi) [inline README_fr.md:2]: It seems like the Markdown format for the 'English' part is incorrect. Also, )I apologize if this upsets you, but) I'd like to arrange it in the order of 'English - French - Japanese'.

```suggestion
[ [**Anglais**](./README_en.md) | Français | [**Japonais**](./README.md) ]
```

Comment (AkimParis) [reply README_fr.md:2]: Yes, please. I was confused about how to include hyperlinks. I don't really care about the order. : )

**Review by kaisugi — APPROVED** [review]
LGTM!

Thank you so much for handling the translation into French. 
This certainly helps to further spread the recognition of Japanese LLMs across the world.

**Pull request merged**